docs/5-simple-functions.markdown

```diff
@@ -113,7 +113,7 @@ this means evaluate `<* .X .X>` in a "world" where `X` is `SET` to
 `5`.
 3. Throw away the "world" created, and restore the `LVAL`s of all 
 `ATOM`s bound in this application of the `FUNCTION ` to their
-originals (if any). This is called "unbinding". -- In ths above, this
+originals (if any). This is called "unbinding". -- In the above, this
```

Comment:
In the original, the parts set off with "--" appeared as paragraphs of their own. Can we do that here?